src/CarbonAware.LocationSources/src/Configuration/LocationDataSourcesConfiguration.cs

```diff
@@ -8,6 +8,6 @@ public class LocationDataSourcesConfiguration
 
     public const string Key = "LocationDataSourcesConfiguration";
 
-    public List<LocationDataSource>? LocationDataSources { get; set; }
+    public List<LocationSourceFile> LocationDataSources { get; set; } = new();
```

Comment:
NIT: var name updated to match

```suggestion
    public List<LocationSourceFile> LocationSourceFiles { get; set; } = new();
```